### PR TITLE
Add x-product property for existing path operations

### PR DESCRIPTION
**Pull request by VladimirBerdnik**
The x-product property allows to filter out paths out of bundled files by decorator

### CODE REVIEW

File: openapi/paths/authentication-options.yaml [hunk 1]
```diff
@@ -1,6 +1,12 @@
 parameters:
   - $ref: ../components/parameters/organizationId.yaml
 get:
+  x-product:
+    - Combined
+    - Full
+    - RiskManagement
+    - Billing
+    - Core
```
[the remark this answers is not in the document]
Comment (adamaltman) [reply openapi/paths/authentication-options.yaml:9]: This is really not part of these APIs.

File: openapi/paths/balance-transactions.yaml [hunk 2]
```diff
@@ -1,6 +1,8 @@
 parameters:
   - $ref: ../components/parameters/organizationId.yaml
 get:
+  x-product:
+    - Users
```
Comment (adamaltman) [inline openapi/paths/balance-transactions.yaml:5]: How can something not be in the Combined and Full? And what is the difference between those?

File: openapi/paths/activation@{token}.yaml [hunk 3]
```diff
@@ -6,6 +6,10 @@ parameters:
     schema:
       type: string
 post:
+  x-product:
+    - Combined
+    - Full
```
Comment (adamaltman) [inline openapi/paths/activation@{token}.yaml:11]: Shouldn't these be for every API? Should we need to type these in?

File: openapi/paths/transactions@{id}@timeline.yaml [hunk 4]
```diff
@@ -2,6 +2,12 @@ parameters:
   - $ref: ../components/parameters/resourceId.yaml
   - $ref: ../components/parameters/organizationId.yaml
 get:
+  x-product:
+    - Combined
+    - Full
+    - Payments
```
Comment (adamaltman) [inline openapi/paths/transactions@{id}@timeline.yaml:8]: We can remove timelines from any of the Payments, Billing, Risk APIs.

File: plugins/products-bundler/mapping/Billing.yaml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+      - Billing Portals
+  - name: Checkout forms
+    tags:
+      - Checkout Forms
+  - name: Customer authentication
+    tags:
+      - Customer Authentication
+  - name: Customers
+    tags:
+      - Tags
```
Comment (adamaltman) [inline plugins/products-bundler/mapping/Billing.yaml:15]:
```suggestion
```

File: plugins/products-bundler/mapping/Billing.yaml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+      - Customers
+      - Customers Timeline
```
Comment (adamaltman) [inline plugins/products-bundler/mapping/Billing.yaml:17]:
```suggestion
```

File: plugins/products-bundler/mapping/Billing.yaml [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+  - name: Orders & invoices
+    tags:
+      - Coupons
+      - Invoices
+      - Orders
+      - Shipping Zones
```
Comment (adamaltman) [inline plugins/products-bundler/mapping/Billing.yaml:23]:
```suggestion
```

File: plugins/products-bundler/mapping/Billing.yaml [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+  - name: Payment Instruments
+    tags:
+      - Payment Instruments
+      - Payment Tokens
+  - name: Payments
+    tags:
+      - Disputes
+      - Fees
+      - Transactions
```
Comment (adamaltman) [inline plugins/products-bundler/mapping/Billing.yaml:32]:
```suggestion
      - Transactions
      - Fees
      - Disputes
```

File: plugins/products-bundler/mapping/Full.yaml [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+      - Customer Authentication
+      - JWT Session
+  - name: Customers
+    tags:
+      - Tags
+      - AML
+      - Customers
+      - Customers Timeline
+      - Files
+      - KYC Documents
```
Comment (adamaltman) [inline plugins/products-bundler/mapping/Full.yaml:45]:
```suggestion
      - Customers
      - Tags
      - Customers Timeline
      - Files
      - KYC Documents
      - AML
```

File: plugins/products-bundler/mapping/Full.yaml [hunk 10]
```diff
@@ -0,0 +1,88 @@
...
+  - name: Gateway accounts
+    tags:
+      - Gateway Accounts
+  - name: Orders & invoices
+    tags:
+      - Coupons
+      - Invoices
+      - Orders
+      - Shipping Zones
```
Comment (adamaltman) [inline plugins/products-bundler/mapping/Full.yaml:54]: This should be with products and pricing.

File: plugins/products-bundler/mapping/RiskManagement.yaml [hunk 11]
```diff
@@ -0,0 +1,24 @@
+info:
+  title: "Risk Management API"
+x-tagGroups:
+  - name: Apps
+    tags:
+      - Experian credentials
+      - Integrations
```
Comment (adamaltman) [inline plugins/products-bundler/mapping/RiskManagement.yaml:7]: Remove
```suggestion
```

File: plugins/products-bundler/mapping/RiskManagement.yaml [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+    tags:
+      - Customer Authentication
+      - JWT Session
+  - name: Customers
+    tags:
+      - AML
+      - Customers
+      - Customers Timeline
+      - Files
+      - KYC Documents
```
Comment (adamaltman) [inline plugins/products-bundler/mapping/RiskManagement.yaml:21]:
```suggestion
      - Customers
      - AML
      - Files
      - KYC Documents
```

File: plugins/products-bundler/mapping/RiskManagement.yaml [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+    tags:
+      - Experian credentials
+      - Integrations
+  - name: Automations
+    tags:
+      - Blocklists
+  - name: Customer authentication
+    tags:
+      - Customer Authentication
+      - JWT Session
```
Comment (adamaltman) [inline plugins/products-bundler/mapping/RiskManagement.yaml:14]:
```suggestion
```

File: plugins/products-bundler/mapping/RiskManagement.yaml [hunk 14]
```diff
@@ -0,0 +1,24 @@
+info:
+  title: "Risk Management API"
+x-tagGroups:
+  - name: Apps
+    tags:
+      - Experian credentials
+      - Integrations
+  - name: Automations
+    tags:
```
Comment (adamaltman) [inline plugins/products-bundler/mapping/RiskManagement.yaml:9]: rules and value lists?

**Comment by VladimirBerdnik** [comment]
Closing this in favor of https://github.com/Rebilly/api-definitions/pull/630

**Pull request closed without merging**